gitextensions/gitextensions
# Added signature info to the commit data panel.

Fixes #3161  .

Changes proposed in this pull request:
 - Commit dialog shows gpg signature information, if any


 
Screenshots before
![nosign](https://user-images.githubusercontent.com/842026/28077424-0bf533a6-6662-11e7-80e4-45fa1de3e7ca.PNG)



Screenshot of a commit with signature
![signed](https://user-images.githubusercontent.com/842026/28077451-17806a88-6662-11e7-834d-e2aa38991a44.PNG)

If the commit isn't signed, the screenshot is the same as "before".




How did I test this code:
 - Create a repository and do some commit
 - [setup gpg and git to use the private key](http://mrexodia.cf/guide/2016/05/30/Github-Gpg)
 - do some signed commit
 - select commit and see the difference shown inside "Commit" panel
